book/content/reproducibility/04/barriers.md

```diff
@@ -24,9 +24,9 @@ This chapter outlines some of those barriers, and a couple of suggestions to get
 
 *replace this text with the content of barrier 3*
 
-## Barrier 4
+## Barrier 4 : Is not considered for promotion
```

Comment:
This wasn't clear in the instructions - so sorry about that. As most of the other contributions have removed the numbered part from this sub-heading I think it makes sense to do that here too. The sections are in a very arbitrary order as it is!
```suggestion
## Is not considered for promotion
```